Assignments/stu5/data-access.js

```diff
@@ -66,3 +67,15 @@ exports.getPerson = async () => {
     }
     return retval;
   };
+
+  exports.getBook = async () => {
+    let bookId = 1
```

Comment:
I think you meant to pass `bookId` into this function, right?

Reply:
PR UPDATED